packages/client/src/ext/jwt-simple.ts

```diff
@@ -0,0 +1,213 @@
+/*
+ * jwt-simple
+ *
+ * JSON Web Token encode and decode module for node.js
+ *
+ * Copyright(c) 2011 Kazuhito Hokamura
+ * MIT Licensed
+ */
+
+/**
+ * module dependencies
+ */
+import { base64 } from '@scure/base'
+import crypto from 'crypto'
+
+/**
+ * support algorithm mapping
+ */
+export type TAlgorithm = 'HS256' | 'HS384' | 'HS512' | 'RS256'
+const algorithmMap: Record<string, string> = {
+  HS256: 'sha256',
+  HS384: 'sha384',
+  HS512: 'sha512',
+  RS256: 'RSA-SHA256',
+}
+
+/**
+ * Map algorithm to hmac or sign type, to determine which crypto function to use
+ */
+const typeMap: Record<string, string> = {
+  HS256: 'hmac',
+  HS384: 'hmac',
+  HS512: 'hmac',
+  RS256: 'sign',
+}
+
+/**
+ * expose object
+ */
+export const jwt = module.exports
```

Comment:
We can't use `module.exports` here since we have an ESM build and don't use bundlers to transpile CJS code.

You can replace this with something like below where each property is the necessary encode/decode function used within our code:
```js
export const jwt = {
  decode: decode,
  encode: encode
}
```

Also ideally move this to the end of the file so the logic makes sense of instantiating the individual functions and then exporting several of them as the named export object `jwt`.  Let me know if that doesn't make sense.

Reply:
Follow-up, it looks like the only code we use from the JWT API is the [`decode` function](https://github.com/ethereumjs/ethereumjs-monorepo/blob/52c31c0451e150e3ef773b8de468114cf18900b4/packages/client/src/util/rpc.ts#L16) so you can probably just export that one function (though I guess also export `encode`) and be done with it.

Reply:
we also use `base64urlDecode` in the test file `jwt-simple.spec.ts`

Reply:
done